app/error_handlers.py

```diff
@@ -88,17 +88,16 @@ async def inactive_case(request):
                 client_id=request['client_id'],
                 trace=request['trace'])
     attributes = check_display_region(request)
-    return jinja.render_template('start-expired.html', request, attributes)
+    return jinja.render_template('start-uac-already-used.html', request, attributes)
 
 
-async def ce_closed(request, collex_id):
-    logger.warn('attempt to access collection exercise that has already ended',
+async def survey_closed(request):
```

Comment:
Same as above, ce_closed is more accurate

Reply:
Updated